source/tag-developers/if-tag.md

```diff
@@ -1,92 +1,36 @@
 ---
-layout: default
-title: Tag Developers Guide
+layout: tag-developers
+title: if tag
+parent:
+  title: Tag Reference
+  url: tag-reference.html
 ---
 
 # if
 
+Please make sure you have read the [Tag Syntax](tag-syntax) document and understand how tag attribute syntax works.
 
-Please make sure you have read the [Tag Syntax](tag-syntax.html) document and understand how tag attribute syntax works.
+## Description__
```

Comment:
Looks like a small typo in the If tag generated HTML.  I think `## Description__` should be `## Description` .

The "__" shows up on the page for the staged site.

Reply:
My bad, fixed :)